openapi/paths/storefront/orders@{id}@change-items.yaml

```
@@ -0,0 +1,39 @@
+parameters:
+  - $ref: ../../components/parameters/resourceId.yaml
+post:
+  x-badge: Experimental
+  x-products:
+    - Storefront
+  tags:
+    - Storefront orders
+  summary: Change subscription order items
+  operationId: StorefrontPostOrderItemsChange
+  x-sdk-operation-name: changeItems
+  security:
+    - CustomerJWT: []
+  description: Changes subscription order items and quantities.
+  requestBody:
+    content:
+      application/json:
+        schema:
+          $ref: ../../components/requestBodies/storefront/PostOrderItemsChange.yaml
+    description: Change items request.
+    required: true
+  responses:
+    '200':
+      description: Subscription items changed.
+      headers:
+        Location:
+          $ref: ../../components/headers/Location.yaml
+      content:
+        application/json:
+          schema:
+            $ref: ../../components/schemas/Quote.yaml
```

```suggestion
            $ref: ../../components/schemas/StorefrontQuoteChangeOrder.yaml
```
We don't have it in the main branch yet, so for reference it will be there - https://github.com/Rebilly/api-definitions/pull/1727/files#diff-6c539e56a6df489fb1c28385096acbb7f2706b4389084601e4dba13b8d43d0bd